### ISSUE TITLE
Spanish BA.1 sublineage with S:E340K and further accumulation of S mutations (6 seq)

### ISSUE THREAD

**Issue by silcn** [cov-lineages/pango-designation#666]
In the spirit of #663, I'm proposing another odd-looking BA.1 sublineage with multiple spike mutations. This one is from Catalunya, Spain, earliest sequence 2022-02-17, and has been seen as recently as 2022-05-04. There are 6 sequences.

Defining mutations are G28378T = ORF9b:R32L and G22580A = S:E340K, although in one sequence this has mutated again to 22580C = S:340Q. 
Most of the sequences have other S mutations too:
EPI_ISL_11676002: no more
EPI_ISL_11676007: L371F
EPI_ISL_12704499: R346T, L452M, I670L, D936Y
EPI_ISL_12704500: R346T, F375S, I670M, G885S
EPI_ISL_12704498: R346T, L371F, F375S, I670M, G885S
EPI_ISL_12601652: K340Q, R346T, L371F, F375S, I670M, G885S
Note that L371F is a partial reversion (the Usher tree gets confused by this and reports it as P371S) and F375S is a reversion.

When I first saw this I expected it would be from a chronic infection, but the metadata shows that the sequences are from at least 5 different people, with the two basal sequences EPI_ISL_11676002 and EPI_ISL_11676007 being the only two that could be from the same person. So this may be another case of spread from a chronic infection to go along with #663, though the infection in question might not have been sequenced. S:E340K on top of BA.1 is a known sotrovimab escape mutation and regularly appears in chronic infections as a result. This feels worthy of further investigation; perhaps the Spanish health authorities are already on the case.

<img width="677" alt="Spain_E340K" src="https://user-images.githubusercontent.com/42699509/170257732-e43e8bbb-ec03-44d3-9cf1-252588a074df.png">

[https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_1cf50_e16cc0.json?branchLabel=Spike%20mutations&c=gt-S_340&label=nuc%20mutations:G22580A](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_1cf50_e16cc0.json?branchLabel=Spike%20mutations&c=gt-S_340&label=nuc%20mutations:G22580A)

**Comment by silcn** [comment]
@bitbyte2015 great spot, probably worth a lineage proposal. Unfortunately Denmark doesn't give enough metadata to tell whether some/all of these are from the same patient.

**Comment by silcn** [comment]
A new one from the Spanish cluster: EPI_ISL_13011167, sampled 2022-05-22. Again, a different patient from all the previous sequences. It's on the S:340Q branch with EPI_ISL_12601652 and has an additional S:L822F.

<img width="701" alt="pango666_2" src="https://user-images.githubusercontent.com/42699509/171040864-a486a235-5d65-4501-a845-03805741d2db.png">

[https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_18f68_5028d0.json?branchLabel=Spike%20mutations&c=userOrOld&label=nuc%20mutations:G22580A](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_18f68_5028d0.json?branchLabel=Spike%20mutations&c=userOrOld&label=nuc%20mutations:G22580A)

**Comment by ryhisner** [comment]
This is I believe the third time I've seen E340K—>K340Q, and I feel pretty certain this has to be related to sotrovimab-treatment, which is known to cause escape mutations at spike residue 337 and 340. There are several other examples of P337X/E340X mutations accompanied by multiple additional spike mutations. I've listed a few below, but there are many others. 

BA.1.18 from Tennessee, April 25, EPI_ISL_13046144 
Spike Mutations: P337H, N354D, K444N, V642G. 
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_bac8_fc4640.json?branchLabel=aa%20mutations&c=pango_lineage&label=nuc%20mutations:C22572A

BA.1.17 from Spain, May 2, EPI_ISL_13011226
Spike Mutations: N149D, E340D, K417T, D936H, and S982T.
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_534_fc6ec0.json?branchLabel=aa%20mutations&c=pango_lineage&label=nuc%20mutations:A1521G,G5578A,C5730T,T7236C,C7299T,C18555T,A22007G,A22582C,A22812C,G24368C,T24506A,C27509T,T27568A,C27944T,C28909T

BA.2 from Switzerland (several others on this branch), May 9, EPI_ISL_12956387 
Spike Mutations: E340Q, A522P, K356R. (This one also has five ORFab, an OFR3a, and an ORF8 mutation) 
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_1428f_7ca30.json?branchLabel=aa%20mutations&c=pango_lineage&label=nuc%20mutations:C6033T

There was a prominent study a while back noting that sotrovimab elicited mutations at spike residues 337 and 340, but the authors did not include any additional information on the patient sequences. It seems possible that sotrovimab might be causing additional mutations besides those at 337/340. Immunocompromised patients are, of course, more likely to receive mAb treatment, and this could be the real reason why so many additional spike mutations accompany those at 337/340. But I think it should be looked into regardless. 
https://www.nejm.org/doi/full/10.1056/NEJMc2120219

As a side note, I've noticed that both S:D936H/Y and S:371S/S:375S reversions seem to accompany the E340X mutations a lot, or at least that's my impression. I could be wrong. Someone with better computer skills than me could maybe look into it and see if there's any correlation there or if it's just my imagination.

**Comment by silcn** [comment]
Yes, agree this is quite clearly sotrovimab-induced. There are quite a lot of these E340K/Q with additional S mutations, and many of them show clear signs of being chronic cases: multiple sequences from the same patient, or the presence of the telltale E:T30I which appears a lot in chronic cases but very rarely transmits onwards. Most of the rest are isolated sequences which could well be from chronic patients too.

This lineage, #732 and #733 are interesting because of the onwards transmission (clearly in the community in the latter two cases; this one could potentially still be restricted to a single healthcare facility)
